### ISSUE TITLE
Unit tests are failing

### ISSUE THREAD

**Issue by oveddan** [arunoda/nodemock#16]
Following test methods are failing:

✔ testMultipleEntriesForOneMethod
method call for: 'foo()' with params: (10, 30) was not executed!

✔ testAssertFailsSameMethod
method call for: 'bar()' with params: (10, 30) was not executed!

✔ testAssertOK
method call for: 'bar()' with params: (10, 30) was not executed!

✔ testAssertThrowsOK
method call for: 'bar()' with params: (10) was not executed!

**Comment by arunoda** [comment]
I don't see tests are failing.

**Comment by oveddan** [comment]
You don't see it locally or you don't see it in the example posted message?

**Comment by arunoda** [comment]
Actually these logs are showing what is happens on the behind.
This the test itself get passed :)
Can't you see this ✔

**Comment by oveddan** [comment]
Yes but the check is misleading.  A mocked condition in the test was never invoked, so you get the error message.  In the unit test you should be invoking every set-up mock. The tests themselves are incorrect and masking the error;  they should be failing.

**Comment by arunoda** [comment]
Ok! Now I got it. You give it a fix.
Thanks.
Pull this request too.
